jerry-core/debugger/jerry-debugger.c

```diff
@@ -391,6 +391,27 @@ jerry_debugger_process_message (uint8_t *recv_buffer_p, /**< pointer the the rec
       return true;
     }
 
+    case JERRY_DEBUGGER_EXCEPTION_CONFIG:
+    {
+      JERRY_DEBUGGER_CHECK_PACKET_SIZE (jerry_debugger_receive_exception_config_t);
+      JERRY_DEBUGGER_RECEIVE_BUFFER_AS (jerry_debugger_receive_exception_config_t, exception_config_p);
+
+      if (exception_config_p->enable == 0)
+      {
+        JERRY_CONTEXT (debugger_flags) = (uint8_t) (JERRY_CONTEXT (debugger_flags)
+                                         | JERRY_DEBUGGER_VM_IGNORE_EXCEPTION);
```

Comment:
I don't think this is the right style. Add a local variable if the line is too long.